src/platform/windows/mod.rs

```diff
@@ -133,7 +133,9 @@ impl MediaControls {
     /// Detach the event handler.
     pub fn detach(&mut self) -> Result<(), Error> {
         self.controls.SetIsEnabled(false)?;
-        self.controls.ButtonPressed(None)?;
+        if let Some(button_handler_token) = self.button_handler_token {
+            self.controls.RemoveButtonPressed(button_handler_token)?;
+        }
```

Comment:
I don't know if this is overkill, but the old method of passing in `None` was removed from the library and it seems like this is how you would remove a handler correctly.

Reply:
It seems that's the correct way to do it. It's fine.